### DISCUSSION TITLE
Vote System

### DISCUSSION THREAD

**Discussion by LuxPoseidon** [EveronLife/EveronLife#85] [Ideas]
There is a built in voting system for player to kick a player already.

We should have the ability to vote for thing like changing the time to day, etc.
See SCR_VoterComponent, it is placed on the player controller.

**Comment by Arkensor** [comment]
Another idea would be to allow players to vote for a restart

**Reply by LuxPoseidon** [reply]
Can get gametime with GetGame().GetWorld().GetWorldTime();

**Reply by LuxPoseidon** [reply]
> Another idea would be to allow players to vote for a restart

EVotingType -> RESTART, ///< Restart the world, no value needed

**Comment by MarvinVomberg** [comment]
I think for most RP servers it is not desirable that players can vote to change stuff like time, weather or even restart the server without a staff member creating that poll or turning that feature on.
What I want to add is that the feature must implement a config option to deactivate the whole system and setting limits on how many players have to vote and a threshold when the voting is successful.
